### ISSUE TITLE
`off` should be an intensity option (not a color)

### ISSUE THREAD

**Issue by stevenroose** [wearefractal/msi-keyboard#5]
I have a MSI GS60 and it has Fn keys that allow you to change the intensity of the keyboard directly to the hardware.

So I had a startup script that set the whole keyboard to green, med. Then the Fn keys allowed me to turn the lights off when I didn't need them. (So since the Fn key changes the intensity and it can turn it off, the `intensity` option should be able to do this too.

Now comes the issue: I wanted the lights to be off on startup, allowing me to turn them on using the Fn keys when I needed them. The only way to do this currently is by setting the color to `off` (and the intensity to whatever). However, when I use the Fn keys then, the light will stay of. (This is actually logical, since it changes the intensity, but the color is set to `off`.)

So basically, off needs to be moved from color to intensity and I think it should be possible.
